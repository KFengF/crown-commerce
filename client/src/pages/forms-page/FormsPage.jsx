import React from 'react';
import SignIn from './components/sign-in/SignIn';
import SignUp from './components/sign-up/SignUp';
import { FormPageDiv } from './FormsPageStyles';
/* import './FormsPage.scss'; */

const FormsPage = () => (
    <FormPageDiv>
        <SignIn />
        <SignUp />
    </FormPageDiv>
);

export default FormsPage;