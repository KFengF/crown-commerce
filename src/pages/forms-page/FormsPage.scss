.forms-page {
    display: flex;
    justify-content: center;
    margin: 20px auto;
    flex-wrap: wrap;
}