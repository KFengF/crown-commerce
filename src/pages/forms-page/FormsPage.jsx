import React from 'react';
import SignIn from './components/sign-in/SignIn';
import SignUp from './components/sign-up/SignUp';
import './FormsPage.scss';

const FormsPage = () => (
    <div className="forms-page" >
        <SignIn />
        <SignUp />
    </div>
);

export default FormsPage;