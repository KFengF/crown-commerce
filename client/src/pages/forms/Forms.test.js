import React from 'react';
import { shallow } from 'enzyme';
import FormsPage from './Forms.page';

describe('Forms page', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<FormsPage />);
    });

    it('should render Forms page', () => {
        expect(wrapper).toMatchSnapshot();
    });
});