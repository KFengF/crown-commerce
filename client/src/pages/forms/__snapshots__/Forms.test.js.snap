// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forms page should render Forms page 1`] = `ShallowWrapper {}`;
